src/app_service/service/community_tokens/service.nim

```diff
@@ -132,6 +143,8 @@ QtObject:
       echo "!!! Contract address ", contractAddress
```

Comment:
I'm thinking.. maybe this info should be useful to keep it in logs, not just debug traces.. wdyt?